### DISCUSSION TITLE
Re submitting info

### DISCUSSION THREAD

**Discussion by Tanveerrifu** [community/community#65509] [GitHub Education]
### Hi everyone,

There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact?tags=education-support) for more information. What can I do now?

**Comment by zhangchi25806** [comment]
Same here. No helps even I submitted tickets for more than 1 month

**Reply by BAp3r** [reply]
Same dude, the support team only replied me once. Then my tickets are always been closed with no response.

**Reply by zhangchi25806** [reply]
@BAp3r Sign...I don't know what to do :S
